server/tilesource/test.py

```diff
@@ -62,17 +114,23 @@ def getTile(self, x, y, z):
             s=(0.3 + (0.7 * yFraction)),
             v=(0.3 + (0.7 * yFraction)),
         )
+        rgbColor = tuple(int(val * 255) for val in backgroundColor)
 
         image = Image.new(
             mode='RGB',
-            size=(self.tileSize, self.tileSize),
-            color=tuple(int(val * 255) for val in backgroundColor)
+            size=(self.tileWidth, self.tileHeight),
+            color=(rgbColor if not self.fractal else (255, 255, 255))
```

Comment:
Fundamentally, they are.  The fractal is drawn by filling the file with the fractal color and then drawing the non-fractal parts recursively.  I don't know of a more efficient way to draw this in PIL.